src/api/posts/routes.py

```diff
@@ -38,6 +43,25 @@ def get(topic_id: int) -> list[dict[str, Any]] | None:
             "topic_id": post.topic_id,
         } for post in posts]
 
+    @staticmethod
+    @authorized_access(provide_user=True)
+    def post(user: User, topic_id: int) -> dict[str, Any]:
```

Comment:
Let's change args order to accept route parameters first, and then all injected parameters from decorators.

For now it may seem to has no sense, but when we will create a bigger endpoints with more parameters, such convention will allow us to easy understand where each parameter comes from and not confuse them.